packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -84,8 +84,10 @@ const NotificationsList = ({ notifications, toggleDialog }) => (
 
 const NotificationListWrapper = React.forwardRef(({ notifications, toggleDialog }, ref) => {
     const is_empty = !notifications.length;
+    const { is_pre_appstore } = React.useContext(PlatformContext);
+
     return (
-        <div className='notifications-dialog' ref={ref}>
+        <div className={`notifications-dialog ${is_pre_appstore && 'notifications-dialog--pre-appstore'}`} ref={ref}>
```

Comment:
Did css changes in notification dialog because the dropdown goes to the right too much.